packages/components/nodes/documentloaders/PlainText/PlainText.ts

```diff
@@ -12,6 +13,7 @@ class PlainText_DocumentLoaders implements INode {
     category: string
     baseClasses: string[]
     inputs: INodeParams[]
+    outputs: INodeOutputsValue[]
 
     constructor() {
         this.label = 'Plain Text'
```

Comment:
can we upgrade the `this.version=2.0` for both nodes?

Reply:
done